### PR TITLE
Cron local system tests

### PR DESCRIPTION
**Pull request by bblay**
A cron job for building some local projects, to be run nightly from a dev environment.

We should consider making a ROSE version of this too, with parallel project builds, to speed up manual testing during the day.

### CODE REVIEW

File: run_configs/_cron/build_all_gfortran.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+set -e
```
Comment (MatthewHambley) [inline run_configs/_cron/build_all_gfortran.sh:1]: I also like to set `-u` which treats dereferencing an undefined macro as an error. This is probably not relevant here as no macros are used. Just thought I'd mention it though.

**Review by MatthewHambley — CHANGES_REQUESTED** [review]
This looks fine, just one comment.

File: run_configs/_cron/build_all_gfortran.sh (hunk 1 above)
Comment (bblay) [reply run_configs/_cron/build_all_gfortran.sh:1]: Thanks, I didn't know about that!

**Review by MatthewHambley — APPROVED** [review]
Happy with this.

**Pull request merged**